front/__mocks__/react-intersection-observer.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { useEffect } from 'react';
```

Comment:
I think Brent deleted this so you can remove it, I think.

Reply:
Cool will do :+1: